lib/process-services-cloud/src/lib/task/task-list/services/task-list-cloud.service.ts

```diff
@@ -56,6 +57,76 @@ export class TaskListCloudService extends BaseCloudService implements TaskListCl
         }
     }
 
+    /**
+     * Available from Activiti version 8.7.0 onwards.
+     * Retrieves a list of tasks using an object with optional query properties.
+     *
+     * @param requestNode Query object
+     * @param queryUrl Query url
+     * @returns List of tasks
+     */
+    fetchTaskList(requestNode: TaskListRequestModel, queryUrl?: string): Observable<any> {
+        if (!requestNode?.appName) {
+            return throwError(() => new Error('Appname not configured'));
+        }
+
+        queryUrl = queryUrl || `${this.getBasePath(requestNode.appName)}/query/v1/tasks/search`;
+
+        const queryParams = {
+            maxItems: requestNode.pagination?.maxItems || 25,
+            skipCount: requestNode.pagination?.skipCount || 0,
+            sort: this.buildSortingParam(requestNode.sorting || [])
+        };
+
+        const queryData = this.buildQueryData(requestNode);
+
+        return this.post<any, TaskCloudNodePaging>(queryUrl, queryData, queryParams).pipe(
+            map((response) => {
+                const entries = response.list?.entries;
+                if (entries) {
+                    response.list.entries = entries.map((entryData) => entryData.entry) as any;
+                }
+                return response;
+            })
+        );
+    }
+
+    getTaskListCounter(requestNode: TaskListRequestModel): Observable<number> {
+        if (!requestNode.appName) {
+            return throwError(() => new Error('Appname not configured'));
+        }
+        return this.fetchTaskList(requestNode).pipe(map((tasks) => tasks.list.pagination.totalItems));
+    }
+
+    protected buildQueryData(requestNode: TaskListRequestModel) {
```

Comment:
can we infer return type here?

Reply:
It's just an object - even in the `BaseCloudService.post` method it's `any`